README.md

```diff
@@ -2825,7 +2825,7 @@ Other Style Guides
     ```
 
   <a name="whitespace--computed-property-spacing"></a>
-  - [19.16](#whitespace--computed-property-spacing) Enforce spacing inside of computed property brackets. eslint: [`computed-property-spacing`](https://eslint.org/docs/rules/computed-property-spacing)
```

Comment:
This is correct as-is; "enforce spacing" can mean either "prohibit spacing" or "require spacing".

Reply:
Well, that's my mistake.